github-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import GitHubUsers from "./GitHubUsers";
+import Followers from "./Followers";
+import "./App.css";
+
+class App extends React.Component {
+  state = {};
+
+  componentDidMount() {}
```

Comment:
Small detail: Try to remove unnecessary lines of code like this line to avoid redundancy and confusion later down the line